test/plugin/test_parser_json.rb

```diff
@@ -28,12 +28,12 @@ def test_fall_back_oj_to_yajl_if_oj_not_available
 
       result = @parser.instance.configure_json_parser(:oj)
 
-      assert_equal [Yajl.method(:load), Yajl::ParseError], result
+      assert_equal [JSON.method(:load), JSON::ParserError], result
```

Comment:
You probably want `JSON.method(:parse)`: https://bugs.ruby-lang.org/issues/19528

It's also very marginally faster.

Reply:
Thanks.

Reply:
Thanks byroot!
Let's fix this in another PR.

Reply:
OK, I will separate the PR